tests/simplex/matrix_free_02.cc

```diff
@@ -206,8 +205,7 @@ class PoissonOperator
   {
     matrix_free.template cell_loop<VectorType, VectorType>(
       [&](const auto &data, auto &dst, const auto &src, const auto range) {
-        const auto       i = data.get_cell_active_fe_index(range);
-        FECellIntegrator phi(matrix_free, 0, 0, 0, i, i);
+        FECellIntegrator phi(matrix_free, range);
```

Comment:
I am using now one of the new constructors in this test. I will use the new `FEFaceEvaluation` constructor in #11273.

Reply:
I have integrated the new `FEFaceEvaluation` constructor in #11273. It works very well and makes the code significantly simpler :+1: